### PR TITLE
Add appendices for pdfs, add open ephys course

### LINKED ISSUE #36: Link BioAmp bands to the "BioAmp Bands" usage guide
The links should be created in the entire docs site

### PR DESCRIPTION
**Pull request by bhawnasehgal**
Hey @lorforlinux 

1. I have updated the content for all BioAmp hardware and DIY kits to include the necessary sections in the pdfs.
2. Added ref links wherever possible.
3. Replaced .. image:: directive with .. figure:: directive across the site.
4. Resolved #36 issue.
5. Added the Open Ephys Extracellular electrophysiology acquisition course.

You can check out the changes [here](https://bhawnasehgal.github.io/upsidedownlabs.github.io/).

### CODE REVIEW

**Review by lorforlinux — COMMENTED** [review]
@bhawnasehgal Thank you for doing this, please resolve the comments I have added.

File: kits/diy-neuroscience/basic/index.rst [hunk 1]
```diff
@@ -93,18 +102,21 @@ We have 2 options to measure the EMG signals, either using the gel electrodes or
...
+.. tip:: Visit the complete documentation on how to :ref:`assemble and use the BioAmp Bands <using-bioamp-bands>` or follow the youtube video given below.
+
    **Tutorial on how to use the band:**
 
-.. youtube:: xYZdw0aesa0
-    :align: center
-    :width: 100%
+   .. youtube:: xYZdw0aesa0
+       :align: center
+       :width: 100%
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply kits/diy-neuroscience/basic/index.rst:119]: By adding indentation, the YouTube video is being shown under the **tip** dialog box. Let me know if this works.

File: kits/diy-neuroscience/basic/index.rst [hunk 2]
```diff
@@ -295,6 +320,7 @@ these frequencies.
 Now everything is configured and connected. So close the settings window and start recording EEG signals.
 
 .. figure:: media/bioamp-exg-pill-eeg.*
+    :align: center
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply kits/diy-neuroscience/basic/index.rst:323]: Okay, making the edit.

File: kits/diy-neuroscience/basic/index.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (lorforlinux) [reply kits/diy-neuroscience/basic/index.rst:119]: Okie, just saw the rendered version. Looks good.

**Comment by lorforlinux** [comment]
@bhawnasehgal Thank you for making these changes, I am merging this now.

**Pull request merged**